app/jobs/course/assessment/submission/fetch_submissions_from_koditsu_job.rb

```diff
@@ -3,7 +3,7 @@ class Course::Assessment::Submission::FetchSubmissionsFromKoditsuJob <
   ApplicationJob
   include TrackableJob
   include Rails.application.routes.url_helpers
-  include Course::Assessment::Submission::KoditsuSubmissionsConcern
+  include Course::Assessment::Submission::Koditsu::SubmissionsConcern
 
   protected
 
```

Comment:
`return unless is_koditsu && assessment.updated_at == updated_at` seems to work just as well as  
`return unless is_koditsu && Time.zone.at(assessment.updated_at) == Time.zone.at(updated_at)`

Is there any critical need to use `Time.zone.at` casting?

Reply:
when I tried it, `assessment.updated_at` refers to UTC, but `updated_at` refers to the local time (which is singapore time), render both sides to be not equal and hence the job is never executed

I used `Time.zone.at` to ensure both times are in the same time zone

Reply:
Ok, then you double-check this code: 
https://github.com/Coursemology/coursemology2/blob/80f1e6b565f7f377587e4546b766c09b570d06ff/app/jobs/course/assessment/invite_to_koditsu_job.rb#L13